### DISCUSSION TITLE
Installing the Software

### DISCUSSION THREAD

**Discussion by TrashTaste44** [arongeo/advrc#4] [Q&A] [answered]
im having a bit of an issue installing this on my flipper, how do you recommend i do it?

**Comment by arongeo** [comment, marked as answer]
Hello, when I originally wrote this I don't think the flipper even existed, so I'm not really sure if it could even work on it. This project was only theoretical, I didn't really expect to put it into any actual use. Of course the project is public, so if you want to you can fork it and maybe try implementing flipper support :))

**Reply by TrashTaste44** [reply]
thank you
